### ISSUE TITLE
Replication of Results from KITTI Object Detection network example

### ISSUE THREAD

**Issue by mattjr** [NVIDIA/DIGITS#977]
Hello,

I am having problems replicating the results from the Object Detection network example. I have tried on several DIGITS 4.0 installations on different computers with different arrangements of GPUs (Titan X, Titan X (Pascal) GTX 1080s). Following the instructions step by step as written in the example I created a KITTI dataset using the prepare_kitti_data.py. I then created a model with lr 0.0001 and the ADAM solver. I specify the network from examples/kitti/detectnet_network.prototxt and used the pretrained weights from dl.caffe.berkeleyvision.org/bvlc_googlenet.caffemodel. Running with the network defaults for batch size (attached image) and then also a batch size of 2 and batch accumulation of 5 as shown in the example.

![screenshot_20160815_123916](https://cloud.githubusercontent.com/assets/4543870/17673087/93a84dd4-62ed-11e6-8542-6dfe0e124055.png)

 I have been unable to achieve the shown performane the mAP and recall and precision is consistently 10 to 15 points lower then what is shown in your example graph.

![screenshot_20160815_134010](https://cloud.githubusercontent.com/assets/4543870/17673138/cebb8fda-62ed-11e6-92b8-2d2bee583200.png)

Additionally our loss_coverage starts much higher is it possible we are using a different pretrained model then was used in the example? Can you share the batch size and lr reduction strategy used to generate the results in the example?

Also I'd mention the % accuracy y right axis is very confusing as I thought our network wasn't converging because at first it seems like the mAP number were being plotted on that axis not the Loss axis on the left. 

Any insight would be great.

**Comment by lukeyeager** [comment]
> Additionally our loss_coverage starts much higher is it possible we are using a different pretrained model then was used in the example?

I don't think so. I'm using this one:
https://github.com/BVLC/caffe/blob/rc3/models/bvlc_googlenet/readme.md

> Can you share the batch size and lr reduction strategy used to generate the results in the example?

We don't have any secret sauce for training DetectNet that we're hiding from you. I follow the steps exactly when I do a training. Sometimes it works and sometimes it doesn't - seemingly due to random initializations. Messing with the learning rate is sometimes helpful. And if you ever get a network that makes good progress, you can start with those trained weights for your next training and it will likely help.

> Also I'd mention the % accuracy y right axis is very confusing as I thought our network wasn't converging because at first it seems like the mAP number were being plotted on that axis not the Loss axis on the left.

Yeah I know. Feel free to upvote https://github.com/NVIDIA/DIGITS/issues/900. We had an intern working on a solution (https://github.com/NVIDIA/DIGITS/pull/810), but he ran into problems with the graphing library and it stalled.
